spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/keras/LocallyConnected2D.scala

```diff
@@ -16,7 +16,8 @@
 
 package com.intel.analytics.bigdl.nn.keras
 
-import com.intel.analytics.bigdl.nn.abstractnn.{AbstractModule, DataFormat, TensorModule}
+import com.intel.analytics.bigdl.nn.abstractnn.{AbstractModule, DataFormat}
+import com.intel.analytics.bigdl.nn.{Container => TContainer, LocallyConnected2D => TLocallyConnected2D}
```

Comment:
This file should also be unchanged?

Reply:
I think it's just delete the useless imnport

Reply:
Yes...